packages/trader/src/App/Components/Elements/Media/media-heading.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const MediaHeading = (props: { children: React.ReactNode | string }) => (
+    <div className='media__heading'>{props.children}</div>
+);
```

Comment:
```suggestion
const MediaHeading: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => (
    <div className='media__heading'>{children}</div>
);
```

Or if you don't want to use `React.FC` for some reason you can do like this:

```suggestion
const MediaHeading = ({ children }: React.PropsWithChildren<unknown>) => (
    <div className='media__heading'>{children}</div>
);
```

Reply:
O, I see, thank you👍🏻